source/localizable/smart_cart/_fbs_supplier_object.html.md.erb

```diff
@@ -0,0 +1,31 @@
+#### Supplier object
+
+Name         | Type   | Description
+------------ | ------ | -----------------
+`id`         | String | Supplier id
+`code`       | String | Supplier code
+`name`       | String | Supplier name
+`address`    | String | Supplier address
+`city`       | String | Supplier City
+`area`       | String | Supplier Area
+`zip_code`   | String | Supplier Zip Code
+`phone`      | String | Supplier Phone
+`tax_number` | String | Supplier Tax Number
+`email`      | String | Supplier Email
+
+#### Supplier creation/update options
+
+These are valid values to POST to the corresponding keys when creating or updating fbs suppliers.
+
+Name                              | Type   | Example     | Description
+--------------------------------- | ------ | ----------- | --------------------------------------------------
+`new_supplier.id`                 | String | "abs30220"              | The supplier ID. Can be fetched from the [list of the shop's suppliers](/smart_cart/fbs_api/#retrieve-shop-suppliers)
+`new_supplier.code`               | String | "a_code"                | Supplier Code
```

Comment:
Not exactly part of this PR but supplier code should be unique per shop for SARMED. 
Will we handle this constraint in our systems or add a validation and require unique supplier-code values from merchants?

Maybe we should address this issue before making the documentation of the API public

Reply:
Created a [task](https://phabricator.skroutz.gr/T120897) for this. Will update documentation here accordingly.

Reply:
[fixed](https://github.com/skroutz/developer.skroutz.gr/pull/196/commits/073c46589f1364844a1732b56adefb99436da3d4)